source/server/server_base.cc

```diff
@@ -0,0 +1,1097 @@
+#include "source/server/server_base.h"
```

Comment:
I have a comment above - doing the file move in its own PR is one option (https://github.com/envoyproxy/envoy/pull/30702) else we could have the base class be Server (as it's in server.cc) and have InstanceImpl create the varoius things (in the new file)
have a preference?

Reply:
Oh sorry, I totally missed that comment. Either approach sounds fine to me. No preference.

Reply:
I think after a bit more thought I'm going to rever that move and go the other route.  Largely because the classes already don't match the files, and if we do the 2 PRs folks have to move their includes back and forth.